packages/mgt-components/src/components/mgt-picker/mgt-picker.ts

```diff
@@ -0,0 +1,270 @@
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import { html, TemplateResult } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { MgtTemplatedComponent } from '@microsoft/mgt-element';
+import { strings } from './strings';
+import { fluentCombobox, fluentOption } from '@fluentui/web-components';
+import { registerFluentComponents } from '../../utils/FluentComponents';
+import '../../styles/style-helper';
+
+registerFluentComponents(fluentCombobox, fluentOption);
+
+/**
+ * Web component that allows a single entity pick from a generic endpoint from Graph. Uses mgt-get.
+ *
+ * @fires selectionChanged - Fired when an option is clicked/selected
+ * @export
+ * @class MgtPicker
+ * @extends {MgtTemplatedComponent}
+ */
+@customElement('mgt-picker')
+export class MgtPicker extends MgtTemplatedComponent {
+  protected get strings() {
+    return strings;
+  }
+
+  /**
+   * The resource to get
+   *
+   * @type {string}
+   * @memberof MgtPicker
+   */
+  @property({
+    attribute: 'resource',
+    reflect: true,
+    type: String
+  })
+  public resource: string;
+
+  /**
+   * Api version to use for request
+   *
+   * @type {string}
+   * @memberof MgtPicker
+   */
+  @property({
+    attribute: 'version',
+    reflect: true,
+    type: String
+  })
+  public version: string = 'v1.0';
+
+  /**
+   * Maximum number of pages to get for the resource
+   * default = 3
+   * if <= 0, all pages will be fetched
+   *
+   * @type {number}
+   * @memberof MgtPicker
+   */
+  @property({
+    attribute: 'max-pages',
+    reflect: true,
+    type: Number
+  })
+  public maxPages: number = 3;
+
+  /**
+   * A placeholder for the picker
+   *
+   * @type {string}
+   * @memberof MgtPicker
+   */
+  @property({
+    attribute: 'placeholder',
+    type: String
+  })
+  public placeholder: string;
+
+  /**
+   * Key to be rendered in the picker
+   *
+   * @type {string}
+   * @memberof MgtPicker
+   */
+  @property({
+    attribute: 'key-name',
+    type: String
+  })
+  public keyName: string;
+
+  /**
+   * Entity to be rendered in the picker
+   *
+   * @type {string}
+   * @memberof MgtPicker
+   */
+  @property({
+    attribute: 'entity-type',
+    type: String
+  })
+  public entityType: string;
+
+  /**
+   * The scopes to request
+   *
+   * @type {string[]}
+   * @memberof MgtPicker
+   */
+  @property({
+    attribute: 'scopes',
+    converter: value => {
+      return value ? value.toLowerCase().split(',') : null;
+    },
+    reflect: true
+  })
+  public scopes: string[] = [];
+
+  /**
+   * Enables cache on the response from the specified resource
+   * default = false
+   *
+   * @type {boolean}
+   * @memberof MgtPicker
+   */
+  @property({
+    attribute: 'cache-enabled',
+    reflect: true,
```

Comment:
Should this be refelected?

Reply:
No, removing all reflect:true instances.